pkg/cmd/admin/policy/policy.go

```diff
@@ -64,8 +64,9 @@ func NewCmdPolicy(name, fullName string, f *clientcmd.Factory, out, errout io.Wr
 			},
 		},
 		{
-			Message: "Assign cluster roles to users and groups:",
+			Message: "Create/assign cluster roles to users and groups:",
 			Commands: []*cobra.Command{
+				NewCmdCreateClusterRole(CreateClusterRoleRecommendedName, fullName+" "+CreateClusterRoleRecommendedName, f, out),
```

Comment:
> this should go under the create subtree, not the policy subtree

I agree.  I'd also like to see a pull with the equivalent command upstream for rbac clusterroles.